.github/workflows/integration_test.yaml

```diff
@@ -168,10 +192,32 @@ jobs:
           if [ ! -z ${{ matrix.modules }} ]; then
             module="-k ${{ matrix.modules }}"
           fi
-          tox -e integration -- --model testing $series $module $args ${{ inputs.extra-arguments }}
+          tox -e integration -- --model testing --keep-models $series $module $args ${{ inputs.extra-arguments }}
       - name: Dump logs
         uses: canonical/charm-logdump-action@main
         if: failure()
         with:
           app: ${{ env.CHARM_NAME }}
           model: testing
+      - name: Setting up kubeconfig ENV for Github Chaos Action
+        if: ${{ inputs.chaos-enabled }}
+        run: echo ::set-env name=KUBE_CONFIG_DATA::$(sudo microk8s config | base64 -w 0)
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+
+      - name: Setup Litmus
+        if: ${{ inputs.chaos-enabled }}
+        uses: merkata/github-chaos-actions@master
```

Comment:
Looks great, but is there a reason why we are using the actions from your personal repo instead of having them in a Canonical repo?

Reply:
Also, it seems you have some merge conflicts to solve first

Reply:
Cool, I'm merging main now, thanks!
As for the second part - the actions repo is part of litmuschaos organization and not canonical's, are you suggesting that the fork is done within canonical organization and not within a repo under my user space?